.vitepress/constants/constants.js

```diff
@@ -1,6 +1,9 @@
 const constants = Object.freeze({
   golangVersion: "1.22.2",
 
+  nodeVersion: "21.7.2",
+  yarnVersion: "1.22.19",
```

Comment:
(comment 1/2) note to remember to update this and also the script when versions change